bin/run.sh

```diff
@@ -74,9 +74,7 @@ case $1 in
       bash
     else
       # python manage.py collectstatic --noinput
-      coverage erase
-      coverage run -m pytest "${@:2}"
-      coverage report -m
```

Comment:
The copious coverage output is copious and not something I ever look at. I don't hate coverage, but I don't want to run it all the time.